modules/ref_argocd-appset-delete.adoc

```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="argocd-appset-delete_{context}"]
+= argocd appset delete
+
+Delete `ApplicationSet` resources for your application.
+
+.Command syntax
+[source,terminal]
+----
+argocd appset delete [flags]
+----
+
+.Example - Delete an existing `ApplicationSet` resource using the following command:
+
+[source,terminal]
+----
+argocd appset delete APPSETNAME (APPSETNAME...)
+----
+
+.argocd appset create flags
+[options="header",cols="1,1"]
+|===
+|Flag |Description
+
+|`-h, --help`
+|Help output for the `argocd appset delete` subcommand.
+
+|`-y, –yes`
+|An `ApplicationSet` resource can have many applications as child resources. So, if you delete an `ApplicationSet` resource, it means that an application is also deleted. If you use the `-y` flag, then the command does not prompt for confirmation and deletes each child application resource.
```

Comment:
`An `ApplicationSet` resource can have one or more applications associated to it as child resources.`
`it means applications associated with the Application set are also deleted`
`and inturn deletes all the applications associated to the ApplciationSet`

Reply:
Fixed.